lib/motion/configuration.rb

```diff
@@ -59,16 +59,14 @@ def initialize
       Rails.application.key_generator.generate_key("motion:secret")
     end
 
+    option :revision_paths do
+      require "rails"
+
+      paths
+    end
+
     option :revision do
-      warn <<~MSG # TODO: Better message (Focus on "How do I fix this?")
-        Motion is automatically inferring the application's revision from git.
-        Depending on your deployment, this may not work for you in production.
-        If it does, add "config.revision = `git rev-parse HEAD`.chomp" to your
-        Motion initializer. If it does not, do something else (probably read an
-        env var or something).
-      MSG
-
-      `git rev-parse HEAD`.chomp
+      RevisionCalculator.new(revision_paths: @revision_paths).perform
```

Comment:
We don't know that `@revision_paths` will be set at this point, and we want to prevent it from changing if we use it here. I believe calling the `revision_paths` method instead here will handle both of these issues.